include/arch/arm/arch/32/mode/machine.h

```diff
@@ -392,6 +392,8 @@ static inline void cleanByVA_PoU(vptr_t vaddr, paddr_t paddr)
     asm volatile("mcr p15, 0, %0, c7, c10, 1" : : "r"(vaddr));
 #elif defined(CONFIG_PLAT_TK1)
     asm volatile("mcr p15, 0, %0, c7, c10, 1" : : "r"(vaddr));
+#elif defined(CONFIG_PLAT_RK3288)
```

Comment:
The armv7 plats use the same instruction, so we probably do not need distinguish plats here for v7.

Reply:
I agree.  Really needs to change to a feature-test macro that  encompases all the armv7 platforms.